react-function-components/src/index.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+const CustomButton = <button>Click Me!</button>;
```

Comment:
For you component define a function that returns the `<button>Click Me!</button>` react element.

Here is the example from the docs
![image](https://user-images.githubusercontent.com/7432943/137979489-76f882d3-dc5f-4908-b3db-ec56f898e844.png)